job_spec/AUTORIFT_ITS_LIVE.yml

```diff
@@ -31,18 +31,12 @@ AUTORIFT:
               minLength: 23
               maxLength: 25
               example: S2A_22WEB_20200627_0_L1C
-            - description: The name of the Landsat 7, 8 or 9 Collection 2 granule to process
+            - description: The name of the Landsat 4, 5, 7, 8 or 9 Collection 2 granule to process
               type: string
-              pattern: "^L[ECO]0[789]_"
+              pattern: "^L([CO]0[89]|E07|T0[45])_.*_T2"
```

Comment:
I don't think we want the `_T2` here

Reply:
Ope yea I realized that after submitting it, so now there's an updated string as `"^L([CO]0[89]|E07|T0[45])_L1"`